src/Files.App/App.xaml.cs

```diff
@@ -148,7 +148,7 @@ private IHost ConfigureHost()
 					.AddSingleton<SettingsViewModel>()
 					.AddSingleton<DrivesViewModel>()
 					.AddSingleton<NetworkDrivesViewModel>()
-					.AddSingleton<OngoingTasksViewModel>()
+					.AddSingleton<StatusCenterViewModel>()
```

Comment:
I know the issue called it "Status center", but we haven't actually made any plans to rename it from "Ongoing tasks".

Reply:
If we decide to go through with this, it's probably best to do it after everything else is reviewed. Otherwise, it's hard to see what changed.